drivers/include/dfplayer.h

```diff
@@ -0,0 +1,751 @@
+/*
+ * Copyright 2019 Marian Buschsieweke
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    drivers_dfplayer DFPlayer Mini MP3 Player
+ * @ingroup     drivers_multimedia
+ * @brief       Driver for the DFPlayer Mini MP3 Player
+ *
+ * The DFPlayer Mini is a super cheap MP3 Player that can be controlled via
+ * UART. It can play files from SD card, USB storage devices and NOR flash.
+ * It has an integrated 3W amplifier that can be used to build a standalone
+ * mono speaker. Alternatively a stereo output can be used that can be
+ * directly connected to headphones, but needs a dedicated amplifier for
+ * connecting to passive speakers.
+ *
+ * File System Setup
+ * =================
+ *
+ * The SD card or USB storage devices needs to be formatted as FAT32 and the
+ * MP3 files have to be stored using one of the following naming schemes, if
+ * selecting the files efficiently is required:
+ *
+ * Numbered File Inside Numbered Folder Naming Scheme
+ * --------------------------------------------------
+ *
+ * The MP3 files have to be named with three decimal digits (e.g. `"001.mp3"` or
+ * `"042"`) and must be placed in in folders named with two decimal digits
+ * (e.g. `"01"` or `"99"`). The folder name numbers must be in the range 1-99
+ * and the file name numbers in the range 1-255. (E.g. `"09/099.mp3"` would be
+ * a valid path.) Playback of these files can be started using the function
+ * @ref dfplayer_play_file
+ *
+ * Numbered File Inside MP3 Folder Naming Scheme
+ * ---------------------------------------------
+ *
+ * The MP3 files have to be named with four decimal digits (e.g. `"0001.mp3"` or
+ * `"1337.mp3") and must be placed in the folder `"MP3"`. The file numbers
+ * 1-9999 are valid. Playback of these files can be started using the function
+ * @ref dfplayer_play_from_mp3
```

Comment:
Can you add a note that characters after the numbers are ignored?
I've seen this in other tutorials about the DFPlayer and it makes working with those files much easier.
(And that this is not true for folders)